### PR TITLE
Fix: some components which offered by @blokhin

### PR DESCRIPTION
**Pull request by GlistenSTAR**

### CODE REVIEW

File: src/lib/components/PeriodicTable/PeriodicTable.svelte [hunk 1]
```diff
@@ -43,35 +43,12 @@
...
-		['Cu/Au', 'group 11'],
-		['Zn/Hg', 'group 12'],
-		['B/Tl', 'triels'],
-		['C/Pb', 'tetrels'],
-		['N/Bi', 'pnictogen'],
-		['O/Po', 'chalcogen'],
-		['F/At', 'halogen'],
-		['He/Rn', 'noble gas'],
-	]);
+	let group_names: any = {};
```
Comment (blokhin) [inline src/lib/components/PeriodicTable/PeriodicTable.svelte:46]: @GlistenSTAR TS offers us `Map<string, string>`, please, let's use it.

**Review by blokhin — CHANGES_REQUESTED** [review]
Please, adopt for `Map`

File: src/lib/components/PeriodicTable/PeriodicTable.svelte (hunk 1 above)
Comment (GlistenSTAR) [reply src/lib/components/PeriodicTable/PeriodicTable.svelte]: > Please, adopt for `Map`

Done

**Review by blokhin — APPROVED** [review]

**Comment by blokhin** [comment]
@GlistenSTAR many thanks :+1: :+1: :+1:

**Pull request merged**